cafebabel/static/js/articles-tags.js

```diff
@@ -2,13 +2,15 @@ class Tags {
   constructor() {
     const context = document.querySelector('.tags')
     this.list = context.querySelector('.tags-list')
-    this.buttonAdd = context.querySelector('input[name=tag-new]')
+    this.fieldAdd = context.querySelector('input[name=tag-new]')
+    this.buttonAdd = context.querySelector('button.add')
+    this.removeButtons = context.querySelectorAll('.tags-list li button')
```

Comment:
`querySelectorAll` returns a _NodeList_.
In some browsers, `forEach` is not implemented on a NodeList.
It may be safer to convert it into an _array_: `this.removeButtons = Array.from(context.querySelectorAll('.tags-list li button'))`